gradle/libs.versions.toml

```diff
@@ -10,6 +10,8 @@ dagger = "2.46.1"
 lifecycle = "2.6.1"
 dokka = "1.8.20"
 mavenPublish = "0.25.3"
+nibelConventionPlugin = "ignored"
+kotlinParcelize = "1.8.21"
```

Comment:
we can use just `kotlin` version here. Kotlin parcelize is part of Kotlin Gradle Plugin, so it shares a general Kotlin version with the rest of plugins.

Reply:
You're right, nice catch!

Reply:
You're right, nice catch!